### DISCUSSION TITLE
The flickering issue that occurs during model rendering.

### DISCUSSION THREAD

**Discussion by QianSunCN** [iTwin/itwinjs-core#7346] [Q&A] [answered]
Hi, is there any way to alleviate the flickering issue that occurs during model rendering?

https://github.com/user-attachments/assets/e194dde0-ebe7-41ff-9c41-ef1cee65e72c

**Comment by pmconne** [comment, marked as answer]
That's called [z-fighting](https://en.wikipedia.org/wiki/Z-fighting). The best way to alleviate it is to eliminate the co-planar geometry.

**Reply by QianSunCN** [reply]
> That's called [z-fighting](https://en.wikipedia.org/wiki/Z-fighting). The best way to alleviate it is to eliminate the co-planar geometry.

Thanks, without changing the model, does iTwinJS have a solution?

**Reply by pmconne** [reply]
No.

**Reply by bbastings** [reply]
If the coincident geometry has some way that it can be uniquely identified, such as by category, you could turn off the category display or explicitly hide elements. Doubtful this is the case judging from your quick video though.
